libcnb-package/src/buildpack_dependency_graph.rs

```diff
@@ -36,7 +36,7 @@ pub fn build_libcnb_buildpacks_dependency_graph(
                 .filter(|buildpack_directory| {
                     matches!(
                         determine_buildpack_kind(buildpack_directory),
-                        Some(BuildpackKind::LibCnbRs | BuildpackKind::Meta)
+                        Some(BuildpackKind::LibCnbRs | BuildpackKind::Composite)
```

Comment:
since we're renaming, i wonder if `LibCnbRs` should be renamed to something like `CompilableComponent` or `LibCnbComponent`.  not necessary but the current name looks like an unpronounceable string of consonants 😅

Reply:
Yeah I'm not too keen on the current name. I've filed #683 to track this separately, since it's separate from this rename, and we'll need to figure out a better name (I like your ideas, but since there are several viable options including in this PR might hold it up whilst we pick one).

Reply:
+1 on what Ed said. I remember having my issue with that name when I did the refactor recently, but couldn't come up with a better name on the spot.